### DISCUSSION TITLE
lvim.colorscheme not respected

### DISCUSSION THREAD

**Discussion by gwww** [LunarVim/LunarVim#3551] [General]
I use Nightfox, installed as so in my `config.lua`:
```
lvim.plugins = {
  {
    "EdenEast/nightfox.nvim",
    run = ":NightfoxCompile",
    event = "BufRead",
  },
}
```

I select NightFox with `lvim.colorscheme = "nightfox"`

When I start lunarvim my colorscheme is `lunar`.

As a test, when I set `lvim.colorscheme = "default"` lunarvim starts with the colorscheme of default.

**Comment by LostInTheLogs** [comment]
use 
```lua
lvim.plugins = {
  {
    "EdenEast/nightfox.nvim",
    run = ":NightfoxCompile",
  },
}
```
with `event = "BufRead"` the colorsheme will be loaded too late
